src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,33 @@
 import React from 'react';
+import './PostInfo.scss';
+import { FullPost } from '../../types';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+interface Props {
+  post: FullPost;
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
```

Comment:
If text `posted by` rendered only for userinfo maybe would be better to move it inside `UserInfo` component ?

Why are you rendering component user when it doesn't exist ?
Let's add check for `user` here before render it.

```suggestion
        {user && <UserInfo user={post.user} />}
```